csrankings-j.csv

```diff
@@ -1461,6 +1461,7 @@ Joël M. H. Karel,Maastricht University,https://dke.maastrichtuniversity.nl/joel
 Joel Moses,Massachusetts Institute of Technology,https://esd.mit.edu/Faculty_Pages/moses/moses.htm,NOSCHOLARPAGE
 Joël Ouaknine [SWS],Max Planck Society,https://people.mpi-sws.org/~joel,ZfE4iQ4AAAAJ
 Joel P. Ilao,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/faculty-profile/single/?id=32742738160&command=GETPROFILE,gd8W9ecAAAAJ
+Joël Porquet-Lupine,Univ. of California - Davis,https://cs.ucdavis.edu/directory/joel-porquet-lupine,hyHs8joAAAAJ

```

Comment:
Please clarify how this faculty member meets the inclusion criteria (tenure-track). I know that at least one UC school (UCSD) has a tenure track for teaching faculty; please verify that this is the case for Davis. Thanks.

Reply:
Hi, the tenure-track teaching faculty series exists across all UC campuses. UCSD (and UCB) may have been a little bit ahead some years ago but UC Davis now has 50+ professors of teaching on our campus.

Here is a document from UC Riverside explaining the professor of teaching series: https://efileinfo.ucr.edu/sites/default/files/2023-02/ucr-guidelines-for-professor-of-teaching-series_7.27.21.pdf. It mentions that "Professor of Teaching titles are **tenure-track positions** designed to meet the long-term instructional needs of the
University that cannot be best fulfilled by an appointee in the regular professorial series."

Hope this clarifies your concerns.

Reply:
Thanks. Missing piece: can you solely advise PhD students in CS? This is a requirement.